components/sendgrid/actions/add-or-update-contact/add-or-update-contact.js

```diff
@@ -0,0 +1,138 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-add-or-update-contact",
+  name: "Add Or Update Contact",
+  description: "Adds or updates a contact.",
```

Comment:
All actions need to include a link to the relevant documentation in the description just like you have in `add-email-to-global-supression.js`